### DISCUSSION TITLE
Scale parameter in `random.exponential()`

### DISCUSSION THREAD

**Discussion by EngineerKhan** [jax-ml/jax#8151] [Ideas]
Like [Numpy's version](https://numpy.org/doc/stable/reference/random/generated/numpy.random.exponential.html), there should be a scale (β) provision in `jax.random.exponential()` as well.

**Comment by jakevdp** [comment]
We've opted not to have an extra parameter, because it's quite easy to scale the results yourself. For example, if you write this in numpy:
```python
x = np.random.exponential(scale=10, size=(1000,))
```
the equivalent in JAX is this:
```python
x = 10 * jax.random.exponential(key, shape=(1000,))
```

**Reply by EngineerKhan** [reply]
The exponential is defined as ​​​λ*e^-​​​λx. So just multiplying by ​​​λ won't do. I have confirmed it too:

```
Z = 10*random.exponential(key,(100,100))
print(X.mean(0).mean(0))
```
And it prints something in the range of 1.00 to 1.10 no matter what do I set the value of λ there.

**Reply by jakevdp** [reply]
That is the formula for the PDF of the exponential – for the sampled values it's a simple scaling:
```python
from jax import random
key = random.PRNGKey(0)
x = 10.0 * random.exponential(key, shape=(100000,))
print(x.mean(), x.std())
# 10.040896 10.021657
```

**Reply by EngineerKhan** [reply]
Oh my bad. One question: Can you please explain that why does it show a whole matrix whenever I call the `x.mean()` there (no matter which distribution):

```
X = 10.0*random.exponential(key,(100,100))
print("The matrix's mean is: ",Z.mean)
```

**Reply by jakevdp** [reply]
You are writing `Z.mean` instead of `Z.mean()`; the former displays information about the `mean` method, which often includes the full representation of the array or matrix:
```python
>>> x = jnp.arange(4)                                                                                                                          
>>> x.mean                                                                                                                                     
<bound method mean of DeviceArray([0, 1, 2, 3], dtype=int32)>
```
The latter will compute the mean:
```python
>>> x.mean()                                                                                                                                   
DeviceArray(1.5, dtype=float32)
```
